### PR TITLE
Redo emoji system

### LINKED ISSUE #65: assets: zopieux must fix his shit
> ###### Originally posted 2016-04-02 15:17 by [serialk](https://bitbucket.org/serialk/) on Bitbucket

    17:15 <zopieux> oublie les assets
    17:15 <zopieux> c'est crade
    17:15 <zopieux> je dois fix cette merde

### PR DESCRIPTION
**Pull request by cafehaine**
Font fallback is kinda not working because of roboto, but otherwise eveything's fine
Fixes #65

### CODE REVIEW

File: prologin/forum/forms.py [hunk 1]
```diff
@@ -17,8 +17,9 @@ class Meta:
             'content': '',
         }
         widgets = {
-            'content': forms.Textarea(attrs={
+            'content': forms.TextInput(attrs={
```
Comment (zopieux) [inline prologin/forum/forms.py]: What.

Comment (cafehaine) [reply prologin/forum/forms.py]: I'll fix this, at the time I pushed this commit I didn't get emoji-picker to work on any textarea

File: prologin/prologin/templates/prologin/base.html [hunk 2]
```diff
@@ -107,6 +108,21 @@
   {# Scripts #}
   <script src="{% static 'lib/js/jquery.min.js' %}"></script>
   <script src="{% static 'lib/js/bootstrap.min.js' %}"></script>
+  <!-- Emoji Picker stuff -->
+  <script src="{% static 'lib/js/config.js' %}"></script>
+  <script src="{% static 'lib/js/util.js' %}"></script>
+  <script src="{% static 'lib/js/jquery.emojiarea.js' %}"></script>
+  <script src="{% static 'lib/js/emoji-picker.js' %}"></script>
```
Comment (zopieux) [inline prologin/prologin/templates/prologin/base.html]: Holly molly that's a lot of Javascripts. Is all that stuff actually necessary?

Comment (cafehaine) [reply prologin/prologin/templates/prologin/base.html]: I'll check, but that was from their exemple page

**Comment by cafehaine** [comment]
Ready for review

File: prologin/forum/static/css/forum.css [hunk 3]
```diff
@@ -215,3 +225,13 @@
   }
 
 }
+
+/* Emoji Picker fix */
+
+div.form-group div.col-sm-10.col-lg-10 i.emoji-picker-icon.emoji-picker {
```
Comment (zopieux) [inline prologin/forum/static/css/forum.css]: Don't include responsive bootstrap stuff here.

Comment (cafehaine) [reply prologin/forum/static/css/forum.css]: It's specifically meant to target a bootstrap thing, as the padding isn't the same on this specific configuration and "breaks" the new thread page, should I add a "new-thread-fix" class instead ?

**Pull request merged**